cmd/apiserver_receive_adapter/main.go

```diff
@@ -173,7 +173,7 @@ func main() {
 	}
 
 	a := apiserver.NewAdaptor(cfg.Host, client, eventsClient, loggerSugared, opt, reporter, env.Name)
-	logger.Info("starting kubernetes api adapter.", zap.Any("adapter", env))
+	logger.Info("starting kubernetes api adapter", zap.Any("adapter", env))
```

Comment:
Leaving the period is fine. In fact, I prefer complete sentences.

Reply:
I added back the period